RemoveVSDirFromCsProj/RazzlePropertyRemover.cs

```diff
@@ -29,6 +29,36 @@ internal class RazzlePropertyRemover
                 " KeepDuplicates=\"false\"",
                 "<Nonshipping>true</Nonshipping>",
                 "<Nonshipping>false</Nonshipping>",
+                "<GenerateAssemblyRefs>true</GenerateAssemblyRefs>",
+                "<SignAssemblyAttribute>true</SignAssemblyAttribute>",
+                @"<?xml version=""1.0"" encoding=""utf-8""?>",
+                @"<Import Project=""$(BuildPropsFile)"" Condition=""'$(BuildProps_Imported)'!='True' AND Exists('$(BuildPropsFile)')"" />",
+                @"<Import Project=""..\Platform.Settings.targets"" />",
+                @"<Import Project=""$(PlatformPath)\Tools\Targets\Platform.Settings.Selector.targets"" />",
+                "<!--Import the targets-->",
+                @"<Import Project=""$(PlatformPath)\Tools\Targets\Platform.Imports.targets"" />",
+                @"<BuildPropsFile>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), Build.props))\Build.props</BuildPropsFile>",
+                @"<OutputPath>$(BinariesDirectory)\bin\$(BuildArchitecture)</OutputPath>",
+                @"<Import Project=""$(PartitionExports)"" />",
+                @"<GeneratedModuleVersion>15.0.0</GeneratedModuleVersion>",
+
+            };
+
+        private static Dictionary<string, string> StringsToReplace { get; } =
+            new Dictionary<string, string>()
+            {
+                {
+                    @"<Project ToolsVersion=""4.0"" DefaultTargets=""Build"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">",
+                    @"<Project Sdk=""Microsoft.NET.Sdk"">"
+                },
+                {
+                    "<OutputType>Library</OutputType>",
+                    "<TargetFramework>net45</TargetFramework>"
+                },
+                {
+                    "<TargetType>Library</TargetType>",
```

Comment:
Do any projects have both `<TargetType>Library</TargetType>` and `<OutputType>Library</OutputType>`? This would mean there would be two `<TargetFramework>net45</TargetFramework>` added right?

 
--
+                    "<TargetType>Library</TargetType>",
 